src/lambda/wizards/samInitWizard.ts

```diff
@@ -193,19 +209,19 @@ export class CreateNewSamAppWizard extends MultiStepWizard<SamCliInitArgs> {
     }
 
     private readonly RUNTIME: WizardStep = async () => {
-        this.runtime = await this.context.promptUserForRuntime(this.runtime)
+        this.runtime = await this.context!.promptUserForRuntime(this.runtime)
```

Comment:
The constructor makes guarantees that `this.context` is not undefined. Make an explicit member so that assertions are not required throughout the class.